tests/template_test.go

```diff
@@ -63,23 +63,23 @@ var _ = Describe("Templates", func() {
 			return func() {
 				By("Generating VM JSON from the Template via oc-process command")
 				_, err := runOcProcessCommand(templateJsonFile, templateParams, vmJsonFile)
-				Expect(err).ToNot(HaveOccurred())
-				Expect(vmJsonFile).To(BeAnExistingFile())
+				ExpectWithOffset(1, err).ToNot(HaveOccurred())
+				ExpectWithOffset(1, vmJsonFile).To(BeAnExistingFile())
 			}
 		}
 
 		assertCreatedVM := func() func() {
 			return func() {
 				By("Creating VM via oc-create command")
 				out, err := runOcCreateCommand(vmJsonFile)
-				Expect(err).ToNot(HaveOccurred())
+				ExpectWithOffset(1, err).ToNot(HaveOccurred())
 				message := fmt.Sprintf("virtualmachine.kubevirt.io \"%s\" created\n", templateParams.Name)
-				Expect(out).To(Equal(message))
+				ExpectWithOffset(1, out).To(Equal(message))
 
 				By("Checking if the VM exists via oc-get command.")
 				Eventually(func() bool {
```

Comment:
Should be `EventuallyWithOffset` for the same reason.

```go
EventuallyWithOffset(1, func() bool { ... })
```

The same applies to all the other assert functions here.